scripts/src/java/org/oppia/android/scripts/coverage/RunCoverage.kt

```diff
@@ -84,8 +83,10 @@ fun main(vararg args: String) {
   val reportFormat = when (format) {
     "HTML" -> ReportFormat.HTML
     "MARKDOWN", "MD" -> ReportFormat.MARKDOWN
+    "PROTO" -> ReportFormat.PROTO
     else -> throw IllegalArgumentException("Unsupported report format: $format")
   }
+  println("format: $reportFormat")
```

Comment:
If you're keeping this, I suggest making it a bit cleaner, e.g. ``"Using format: $reportFormat."``.

Reply:
Sure, updated with the same.